app/src/main/kotlin/no/nav/k9punsj/korrigeringinntektsmelding/KorrigeringInntektsmeldingService.kt

```diff
@@ -43,24 +50,38 @@ internal class KorrigeringInntektsmeldingService(
 
     private val logger: Logger = LoggerFactory.getLogger(KorrigeringInntektsmeldingService::class.java)
 
-    internal suspend fun henteMappe(norskIdent: String): SvarOmsDto {
+    internal suspend fun henteMappe(norskIdent: String): ServerResponse {
```

Comment:
Burde ikke man prøve å få ut ServerResponse fra serviceklassene?
Det er presentasjonslaget, altså apiet som burde returnere ServerResponse.

Reply:
Jo, det var det jag begynte på her i høst, men hann jo bara fikse OMP_UT och her er fortsatt en del feil... så jag reverter for o få alla ytelser lika.

Nå ska jo bjørn erik & stian ev. begynne o gøra store endringer i punsj så tipper det er fint o standardisere o fikse så mye småbugs som muligt før de kommer igang :)
 
På sikt tipper jag det er enklere o gå vekk fra functional routing o bara bruke restcontrollers ..

Reply:
OK 👍